src/backend/optimizer/plan/plangroupext.c

```diff
@@ -446,6 +442,13 @@ plan_grouping_extension(PlannerInfo *root,
 	context.canonical_rollups = NIL;
 	context.curr_grpset_no = 0;
 
+	if (*p_grpColIdx != NULL)
+		memcpy(context.grpColIdx, *p_grpColIdx,
```

Comment:
And now context.grpColIdx and context.grpOperators can be uninitialized?

Reply:
As I have checked there is one GROUP BY operator for each grouping set, so it makes sense to assert them both, as they both use `*p_numGroupCols` for their size

Reply:
When this code is executed?
https://github.com/arenadata/gpdb/blob/fd5939d4f491238bbd28be88b98949cd0e2d6fe0/src/backend/optimizer/plan/plangroupext.c#L511-L519

Reply:
When there is no grouping keys, eg: 
https://github.com/arenadata/gpdb/blob/cb99eab6234c784b114f4bfe0cefed03c06e379d/src/backend/cdb/cdbgroup.c#L2948-L2953